servicenow_connector.py

```diff
@@ -16,26 +16,25 @@
 #
 # Phantom imports
 import phantom.app as phantom
+
 try:
     import phantom.rules as phrules
 except:
     pass
-from phantom.base_connector import BaseConnector
-from phantom.action_result import ActionResult
-
-# THIS Connector imports
-from servicenow_consts import *
-
-import sys
-import pytz
 import json
+import re
+import sys
+from datetime import datetime
+
 import magic
+import pytz
 import requests
-from bs4 import BeautifulSoup
-from bs4 import UnicodeDammit
-from datetime import datetime
-import re
+from bs4 import BeautifulSoup, UnicodeDammit
+from phantom.action_result import ActionResult
+from phantom.base_connector import BaseConnector
 
+# THIS Connector imports
```

Comment:
Should we remove this comment? It seems redundant.

Reply:
Yes, I have made the suggested changes.